ts/screens/wallet/payment/ScanQrCodeScreen.tsx

```diff
@@ -194,6 +183,18 @@ class ScanQrCodeScreen extends React.Component<Props, State> {
     this.showImagePicker();
   };
 
+  private handleBarcodeScanned = (barcode: ScannedBarcode) => {
+    if (this.state.scanningState === "SCANNING") {
```

Comment:
we should add another check (this is my proposal)
![Schermata 2022-05-11 alle 10 40 52](https://user-images.githubusercontent.com/822471/167807253-4f2aa4b4-9a39-4729-971b-574568c1594c.png)

This is avoid the marker flickering
how to reproduce? point the camera an generic QrCode, like the one below, and hold on it for at least 20/30 secs
![Schermata 2022-05-11 alle 10 42 48](https://user-images.githubusercontent.com/822471/167808158-ddc43f5f-2685-4159-9630-3aaf4caf0e8b.png)

Reply:
Definitely 🙏 added here: https://github.com/pagopa/io-app/pull/3928/commits/e13c634791a6e9b0e0c724a65af6c650517cc39f